### DISCUSSION TITLE
ユーザーページ RSS における取得件数制限について

### DISCUSSION THREAD

**Discussion by su8ru** [increments/qiita-discussions#204] [要望・アイデア・不具合]
## 現状

4 月上旬以降、ユーザーページの RSS で返ってくる記事が直近 4 件のみになっているようですが、これは意図した変更なのでしょうか？

例: https://qiita.com/su8ru/feed

## 要望

全件取得、あるいは `?limit=25` のようなオプションを指定できるようにしていただきたいです。もし既にあるのであればご教示ください。

**Comment by kyntk** [comment]
@su8ru ご質問いただきありがとうございます。

RSSで取得できる件数は、4/6に4件にする仕様変更を行いました。
変更をした理由としては、パフォーマンス改善の目的です。
仕様変更についてのアナウンスができておらず大変申し訳ございません :bow: 

RSSの機能はRSSリーダーなどで投稿された記事をリアルタイムで確認するための用途として公開しております。
一方、QiitaはAPIも公開しており、こちらはQiitaにあるデータを取得するための用途となっております。

記事を全件もしくは25件などの直近の投稿以外を取得するものはRSSではなく、APIを利用していただきたきますよう、お願いいたします :bow:

**Reply by su8ru** [reply]
ご返答ありがとうございます！

いえいえ、言われてみれば本来の用途とは異なるのかもしれませんね……

当面は API → RSS に変換を掛けつつ様子を見たいと思います。
